README.md

```diff
@@ -2,3 +2,9 @@
  [![Build Status](https://travis-ci.org/ColonialMarines-Mirror/ColonialMarines-2018.svg?branch=master)](https://travis-ci.org/ColonialMarines-Mirror/ColonialMarines-2018)
  
  This is a fork based off the July-2018 version of ColonialMarines
+
+# Copyright disclaimer
+This repo is licensed under AGPLv3, and due to the fact that it was based on the BayStation 12 codebase along with the instance of the goonstation 2016 process scheduler in the codebase without an explicit, written override of the licence from the rights holder overriding the AGPL licence, any attempts to DMCA this repo are in violation of GitHub's policies and constitute perjury under US federal law as lying or misrepresenting facts in a sworn declaration is a felony under 18 U.S.C. § 1621, and after a counternotification is filed may be reported to the FBI and GitHub. 
```

Comment:
its more the goon computer code that isnt certain the goon processes scheduler is possibly extempted in this situation.

Reply:
Link to code?